twitcher/wps_restapi/swagger_definitions.py

```diff
@@ -303,127 +303,60 @@ class OkGetLogsResponse(MappingSchema):
 }
-get_all_jobs_response = {
+get_all_jobs_responses = {
     '200': OkGetAllJobsResponse(description='success')
 }
-get_single_job_status_response = {
+get_single_job_status_responses = {
     '200': OkGetSingleJobStatusResponse(description='success')
 }
-get_single_job_outputs_response = {
+get_single_job_outputs_responses = {
     '200': OkGetSingleJobOutputsResponse(description='success')
 }
-get_single_output_response = {
+get_single_output_responses = {
     '200': OkGetSingleOutputResponse(description='success')
 }
-get_exceptions_response = {
+get_exceptions_responses = {
     '200': OkGetExceptionsResponse(description='success')
 }
-get_logs_response = {
+get_logs_responses = {
     '200': OkGetLogsResponse(description='success')
 }
 
 
-class GetProvider(MappingSchema):
-    querystring = ProviderEndpoint()
+class LaunchJobQuerystring(MappingSchema):
+    sync_execute = SchemaNode(Boolean(), example='application/json', default=False, missing=colander.drop)
+    sync_execute.name = 'sync-execute'
 
 
-class GetProviders(MappingSchema):
-    pass
+class JsonHeader(MappingSchema):
+    content_type = SchemaNode(String(), example='application/json', default='application/json')
+    content_type.name = 'Content-Type'
 
 
 class PostProvider(MappingSchema):
     body = CreateProviderRequestBody()
+    header = JsonHeader()
 
 
-class DeleteProvider(MappingSchema):
-    querystring = ProviderEndpoint()
-
-
-"""
-Processes schemas
-"""
-
-
-class GetProcesses(MappingSchema):
-    # reusing the Provider query because both only have provider_id as field in the query string
-    querystring = ProviderEndpoint()
-
-
-class GetProcess(MappingSchema):
-    querystring = ProcessEndpoint()
-
-
-class PostProcess(MappingSchema):
-    """Launching a new process request definition"""
-    querystring = ProcessEndpoint()
+class PostProcess(ProcessEndpoint):
     body = LaunchJobRequestBody()
-
-
-class GetJobs(MappingSchema):
-    pass
-
-
-class GetJobStatusFull(MappingSchema):
-    querystring = FullJobEndpoint()
-
-
-class GetJobStatusShort(MappingSchema):
-    querystring = ShortJobEndpoint()
-
-
-class DismissJobFull(MappingSchema):
-    querystring = FullJobEndpoint()
-
-
-class DismissJobShort(MappingSchema):
-    querystring = ShortJobEndpoint()
-
-
-class GetJobOutputsFull(MappingSchema):
-    querystring = FullJobEndpoint()
-
-
-class GetJobOutputsShort(MappingSchema):
-    querystring = ShortJobEndpoint()
-
-
-class GetSpecificOutputFull(MappingSchema):
-    querystring = FullOutputEndpoint()
-
-
-class GetSpecificOutputShort(MappingSchema):
-    querystring = ShortOutputEndpoint()
-
-
-class GetExceptionsFull(MappingSchema):
-    querystring = FullExceptionsEndpoint()
-
-
-class GetExceptionsShort(MappingSchema):
-    querystring = ShortExceptionsEndpoint()
-
-
-class GetLogsFull(MappingSchema):
-    querystring = FullLogsEndpoint()
-
-
-class GetLogsShort(MappingSchema):
-    querystring = ShortLogsEndpoint()
```

Comment:
was this delete intended?
seems like they are still used in wps_restapi/jobs/jobs.py